standard/icon_data.lua

```diff
@@ -74,6 +74,7 @@ return {
 	activeroster = 'fas fa-users',
 	history = 'fas fa-books',
 	media = 'fas fa-newspaper',
+	goto = 'far fa-arrow-right',
```

Comment:
Maybe quote that key, as goto is a keyword in newer Lua versions

Reply:
Yeah I will, or change the name